client/my-sites/posts/post.jsx

```diff
@@ -16,9 +16,11 @@ import PostHeader from './post-header';
 import PostImage from '../post/post-image';
 import PostExcerpt from 'components/post-excerpt';
 import PostTotalViews from 'my-sites/posts/post-total-views';
+import CommentButton from 'blocks/comment-button';
 import utils from 'lib/posts/utils';
 import updatePostStatus from 'lib/mixins/update-post-status';
 import analytics from 'lib/analytics';
+import { shouldShowComments } from 'reader/comments/helper';
```

Comment:
Is this logic applicable outside the Reader (and will it remain like that)? The helper function seems to disable comments for Jetpack sites, for instance.